CODEOWNERS

```diff
@@ -1,8 +1,22 @@
-# Default rules
+# Website Development
+* @nodejs/nodejs-website
 
-* @nodejs/website
+# Infrastructure
+.github @nodejs/web-infra
+.husky @nodejs/web-infra
 
-# Node.js Release Blog Posts
+# Framework
+next.config.mjs @nodejs/web-infra
+next.dynamic.mjs @nodejs/web-infra
```

Comment:
Do we want to have shared ownership here? Or are we happy with just infra?

Reply:
I'd rather have web-infra as ownership. That doesn't mean the remainder of the team can open PRs and that these PRs can get merged; it's just that the technical ownership goes to the web infra team.

Reply:
this is the only outstanding question - I suspect we can adjust if it comes to be a problem